src/components/screens/SendScreen/components/TransactionReviewBottomSheet.tsx

```diff
@@ -0,0 +1,169 @@
+import { BigNumber } from "bignumber.js";
+import { AssetIcon } from "components/AssetIcon";
+import Avatar from "components/sds/Avatar";
+import { Button, IconPosition } from "components/sds/Button";
+import Icon from "components/sds/Icon";
+import { Text } from "components/sds/Typography";
+import { PricedBalance } from "config/types";
+import { ActiveAccount } from "ducks/auth";
+import { isLiquidityPool } from "helpers/balances";
+import { formatAssetAmount, formatFiatAmount } from "helpers/formatAmount";
+import useAppTranslation from "hooks/useAppTranslation";
+import useColors from "hooks/useColors";
+import React from "react";
+import { View } from "react-native";
+
+type TransactionReviewBottomSheetProps = {
+  selectedBalance: PricedBalance | undefined;
+  tokenValue: string;
+  address: string;
+  account: ActiveAccount | null;
+  publicKey: string | undefined;
+  onCancel?: () => void;
+  onConfirm?: () => void;
+};
+
+const TransactionReviewBottomSheet: React.FC<
+  TransactionReviewBottomSheetProps
+> = ({
+  selectedBalance,
+  tokenValue,
+  address,
+  account,
+  publicKey,
+  onCancel,
+  onConfirm,
+}) => {
+  const { t } = useAppTranslation();
+  const { themeColors } = useColors();
+  const slicedAddress = `${address.slice(0, 6)}...${address.slice(-4)}`;
+
+  return (
+    <View className="flex-1">
+      <View className="rounded-[16px] p-[24px] gap-[24px] bg-background-secondary">
+        <Text lg medium>
+          {t("transactionReviewScreen.title")}
+        </Text>
+        <View className="gap-[16px]">
+          {selectedBalance && !isLiquidityPool(selectedBalance) && (
+            <View className="w-full flex-row items-center gap-4">
+              <AssetIcon token={selectedBalance} />
+              <View className="flex-1">
+                <Text xl medium>
+                  {formatAssetAmount(tokenValue, selectedBalance.tokenCode)}
+                </Text>
+                <Text md medium secondary>
+                  {formatFiatAmount(
+                    new BigNumber(tokenValue).times(
+                      selectedBalance.currentPrice || 0,
```

Comment:
in all places, if price is `undefined` or `null` then we want to either **display "--" or hide it completely** to avoid having users confused thinking that their tokens is worth zero dollars. Does that make sense?